### PR TITLE
vercel-cli 32.3.1

### PR DESCRIPTION
**Pull request by dnhn**
Created with `brew bump-formula-pr`.

### CODE REVIEW

**Comment by chenrui333** [comment]
```
  dist/index.js:
    expected replacement of "= getUpdateCommand" with "= async()=>'brew upgrade vercel-cli'"
```

**Review by fxcoudert — APPROVED** [review]

File: Formula/v/vercel-cli.rb [hunk 1]
```diff
@@ -33,7 +31,7 @@ def install
                 .each { |dir| dir.rmtree if dir.basename.to_s != "#{os}-#{arch}" }
 
     # Replace universal binaries with native slices
-    deuniversalize_machos
+    deuniversalize_machos libexec/"lib/node_modules/wrangler/node_modules/fsevents/fsevents.node"
```
Comment (chenrui333) [inline Formula/v/vercel-cli.rb:34]:
```
  vercel-cli
    * Binaries built for a non-native architecture were installed into vercel-cli's prefix.
      The offending files are:
        /opt/homebrew/Cellar/vercel-cli/32.3.1/libexec/lib/node_modules/vercel/node_modules/fsevents/fsevents.node	(x86_64)
```

**Comment by iMichka** [comment]
```
2023-09-22T20:39:53.5605280Z npm sill ADD node_modules/vercel/node_modules/@vercel/node/node_modules/async-listen
2023-09-22T20:39:53.5605360Z npm sill ADD node_modules/vercel/node_modules/@vercel/next
2023-09-22T20:39:53.5605440Z npm sill ADD node_modules/vercel/node_modules/@vercel/hydrogen
2023-09-22T20:39:53.5605520Z npm sill ADD node_modules/vercel/node_modules/@vercel/go
2023-09-22T20:39:53.5605610Z npm sill ADD node_modules/vercel/node_modules/@vercel/fun
2023-09-22T20:39:53.5605730Z npm sill ADD node_modules/vercel/node_modules/@vercel/build-utils
2023-09-22T20:39:53.5605730Z 
2023-09-22T20:39:53.5605790Z added 248 packages in 12s
2023-09-22T20:39:53.5605790Z 
2023-09-22T20:39:53.5605860Z 18 packages are looking for funding
2023-09-22T20:39:53.5605910Z   run `npm fund` for details
2023-09-22T20:39:53.5605960Z npm verb exit 0
2023-09-22T20:39:53.5606000Z npm info ok 
2023-09-22T20:39:53.5606120Z [31mError:[0m An exception occurred within a child process:
2023-09-22T20:39:53.5614700Z   Errno::ENOENT: No such file or directory @ rb_file_s_stat - /opt/homebrew/Cellar/vercel-cli/32.3.1/libexec/lib/node_modules/wrangler/node_modules/fsevents/fsevents.node
2023-09-22T20:39:53.5615070Z ##[endgroup]
2023-09-22T20:39:53.5615300Z ##[group]Truncated install --build-bottle vercel-cli output
2023-09-22T20:39:53.5651410Z ##[error]npm sill ADD node_modules/vercel/node_modules/@jridgewell/sourcemap-codec
```

**Comment by chenrui333** [comment]
#151959

**Pull request closed without merging**